<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapHoverInfoWidget</class>
 <widget class="QWidget" name="MapHoverInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>145</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>悬停信息</string>
  </property>
  <layout class="QVBoxLayout" name="widgetLayout">
   <item>
    <widget class="QGroupBox" name="enableHoverInfoGroupBox">
     <property name="title">
      <string>启用悬停信息</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="groupBoxLayout">
      <item>
       <widget class="QCheckBox" name="showNamesCheckBox">
        <property name="text">
         <string>显示名称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showLabelsCheckBox">
        <property name="text">
         <string>显示项目标签</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MapHoverInfo::ListSelectionWidget" name="platformHoverInfo" native="true"/>
      </item>
      <item>
       <widget class="MapHoverInfo::ListSelectionWidget" name="trackHoverInfo" native="true"/>
      </item>
      <item>
       <widget class="MapHoverInfo::ListSelectionWidget" name="platformEventHoverInfo" native="true"/>
      </item>
      <item>
       <widget class="MapHoverInfo::ListSelectionWidget" name="weaponEventHoverInfo" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapHoverInfo::ListSelectionWidget</class>
   <extends>QWidget</extends>
   <header>MapHoverInfoListSelectionWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
